### DISCUSSION TITLE
Support for door open/closed sensor

### DISCUSSION THREAD

**Discussion by isaackehle** [jbergler/hass-ttlock#133] [Ideas]
Hi there,
Have you considered supporting an open/closed door contact sensor? 
If there were a way I could look into this, as I know Python, I'd be ok taking a look. Programming for HA is new to me, so being pointed in the right direction would be needed.

Thanks,
Isaac

**Comment by jbergler** [comment]
Hey Isaac, there's an issue with some discussion about what this might look like #72 
I'd be happy to accept a PR - for me the hardest part here is testing since I don't have a lock with a sensor.

I wrote out some steps, hopefully that helps you get started.

**Reply by isaackehle** [reply]
Cool, I searched for an existing thread but it can't up with nothing. 

I do have a sensor, and it essentially pairs with the lock as a single device. So hopefully it's a matter of checking the logs after triggering a state change. 

I'll take a look at the thread and see what I can do.
